man/examples-code-check.c

```diff
@@ -607,6 +607,8 @@ main(int argc, char *argv[]) {
             continue;
           }
           memcpy(name_list[i], buffer, sizeof(name_list[i])-1);
+          /* CID 1566472 */
```

Comment:
I think this is a false positive as the last byte is never overwritten and therefore has the initial zero.
Anyway, setting the value explicitly does not cause any harm.
In addition, I suggest using `strncmp` instead of `strcmp` as it is the real pitfall here.

Reply:
Used strncmp()